wolfcrypt/src/aes.c

```diff
@@ -9914,22 +9914,13 @@ static WARN_UNUSED_RESULT int wc_AesFeedbackEncrypt(
     Aes* aes, byte* out, const byte* in, word32 sz, byte mode)
 {
     byte*  tmp = NULL;
-#ifdef WOLFSSL_AES_CFB
```

Comment:
As discussed odd seeing this removed. Sounds like this is dead code after a refactor.

Reply:
right, dead code from refactor.  the changes here just complete the refactor to avoid do-nothing code and `clang-analyzer-deadcode.DeadStores`.